<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>845</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tablePasswords">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>121</y>
     <width>911</width>
     <height>491</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView {
    background-color: #1e1e1e;        /* —Ç—ë–º–Ω—ã–π —Ñ–æ–Ω —Ç–∞–±–ª–∏—Ü—ã */
    border: 1px solid #3a3a3a;        /* –ª—ë–≥–∫–∞—è —Ä–∞–º–∫–∞ */
    gridline-color: #2f2f2f;          /* —Ü–≤–µ—Ç —Ä–∞–∑–¥–µ–ª–∏—Ç–µ–ª—å–Ω—ã—Ö –ª–∏–Ω–∏–π */
    color: #dcdcdc;                   /* —Ü–≤–µ—Ç —Ç–µ–∫—Å—Ç–∞ */
    selection-background-color: #3d3d3d;   /* —Ü–≤–µ—Ç –≤—ã–¥–µ–ª–µ–Ω–∏—è —Å—Ç—Ä–æ–∫–∏ */
    selection-color: #ffffff;         /* —Ç–µ–∫—Å—Ç –ø—Ä–∏ –≤—ã–¥–µ–ª–µ–Ω–∏–∏ */
    outline: 0;                        /* —É–±–∏—Ä–∞–µ–º —Å–∏–Ω—é—é —Ä–∞–º–∫—É */
	alternate-background-color: #262626;

}

QTableView::item {
    padding: 6px;                     /* –≤–Ω—É—Ç—Ä–µ–Ω–Ω–∏–µ –æ—Ç—Å—Ç—É–ø—ã */
}

/* –£–±–∏—Ä–∞–µ–º —Å–∏–Ω—é—é —Ä–∞–º–∫—É –≤–æ–∫—Ä—É–≥ —Ç–µ–∫—É—â–µ–π —è—á–µ–π–∫–∏ */
QTableView::item:focus {
    outline: none;
}

/* –ü–æ–¥—Å–≤–µ—Ç–∫–∞ —Å—Ç—Ä–æ–∫–∏ –ø—Ä–∏ –Ω–∞–≤–µ–¥–µ–Ω–∏–∏ */
QTableView::item:hover {
    background-color: #2a2a2a;
}

/* –ó–∞–≥–æ–ª–æ–≤–æ–∫ —Ç–∞–±–ª–∏—Ü—ã */
QHeaderView::section {
    background-color: #252525;
    color: #dcdcdc;
    padding: 6px;
    border: none;
    border-right: 1px solid #3a3a3a;
    font-weight: bold;
}

/* –£–±–∏—Ä–∞–µ–º –≥—Ä–∞–Ω–∏—Ü—É –ø–æ—Å–ª–µ–¥–Ω–µ–π —Å–µ–∫—Ü–∏–∏ –∑–∞–≥–æ–ª–æ–≤–∫–∞ */
QHeaderView::section:last {
    border-right: none;
}

/* –í–µ—Ä—Ç–∏–∫–∞–ª—å–Ω—ã–µ –∑–∞–≥–æ–ª–æ–≤–∫–∏ (–æ–±—ã—á–Ω–æ —Å–∫—Ä—ã—Ç—ã, –Ω–æ –Ω–∞ –≤—Å—è–∫–∏–π —Å–ª—É—á–∞–π) */
QHeaderView::section:vertical {
    border-bottom: 1px solid #3a3a3a;
}

/* –í—ã—Ä–∞–≤–Ω–∏–≤–∞–Ω–∏–µ —Ç–µ–∫—Å—Ç–∞ –∑–∞–≥–æ–ª–æ–≤–∫–æ–≤ –ø–æ —Ü–µ–Ω—Ç—Ä—É */
QHeaderView::section {
    text-align: center;
}

/* –°—Ç–∏–ª—å –ø–æ–ª–æ—Å—ã –ø—Ä–æ–∫—Ä—É—Ç–∫–∏ */
QScrollBar:vertical {
    background: #1e1e1e;
    width: 10px;
    margin: 0px;
}

QScrollBar::handle:vertical {
    background: #444444;
    min-height: 20px;
    border-radius: 4px;
}

QScrollBar::handle:vertical:hover {
    background: #5a5a5a;
}

QScrollBar::add-line,
QScrollBar::sub-line {
    background: none;
    border: none;
    height: 0px;
}



</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>814</x>
     <y>90</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add (F4)</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>241</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::AutoText</enum>
   </property>
   <property name="pixmap">
    <pixmap/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>640</y>
     <width>871</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: #1f1f1f;
    border: 1px solid #333;
    border-radius: 10px;
    padding: 14px;
    color: #cfcfcf;
    font-size: 11pt;
    font-family: Segoe UI, Roboto, sans-serif;
}
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;How to use Password Keeper&lt;/span&gt;&lt;br/&gt;&lt;br/&gt;üîê &lt;span &gt;Add &lt;/span&gt;‚Äî create a new password entry.&lt;br/&gt;üìã Your saved passwords appear in the table above.&lt;br/&gt;‚úèÔ∏è Selecting a row and you can edit it.&lt;br/&gt;üóëÔ∏è By selecting a row you can also delete it.&lt;br/&gt;üíæ All your data is stored securely and locally.&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextInteractionFlag::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QPushButton" name="delButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>90</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Delete (F2)</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>90</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Edit (F1)</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Search (F3)</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetMasterPassword_btn">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>782</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reset master password</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
